### PR TITLE
Convert transforms to 3D

### PR DESCRIPTION
**Pull request by cyVR**
Short description explaining the high-level reason for the pull request

## Additions

-

## Removals

-

## Changes

-

## Testing

-

## Notes

-

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Assignee tool](https://help.github.com/articles/assigning-issues-and-pull-requests-to-other-github-users/) :arrow_right:

### CODE REVIEW

**Comment by sebworks** [comment]
@anselmbradford, was there a reason we didn't do this originally? I can't remember. Android, Blackberry, etc...

File: cfgov/unprocessed/css/misc.less [hunk 1]
```diff
@@ -670,11 +670,11 @@
     - cfgov-misc
 */
 .u-move-transition {
-    transition: transform 0.25s ease-out;
+    transition: transform 0.25s linear;
 }
 
 .u-alpha-transition {
-    transition: opacity 0.25s ease-out;
+    transition: opacity 0.25s linear;
```
Comment (Scotchester) [inline cfgov/unprocessed/css/misc.less]: The rest of the PR may be helpful, but please revert the changes to easing methods (here and L673). We chose `ease-out` intentionally.

**Comment by jimmynotjim** [comment]
@sebworks @anselmbradford I don't remember either, we must have just missed it. Now that `will-change` [is available](https://developer.mozilla.org/en-US/docs/Web/CSS/will-change), we should consider using that with `3d` as a fallback with an `@supports` query for `will-change`. That could definitely go on the backlog though and merge this once @cyVR reverts the timing change @Scotchester pointed out.

**Comment by sebworks** [comment]
sounds good, 👍

**Comment by Scotchester** [comment]
LOL, I just realized that Ans wrote the original commit for this, and it exists on a `3dtransforms` branch in our repo.

**Comment by anselmbradford** [comment]
Haha, yeah sorry, I don't recall why I didn't PR that original branch but my best guess is that linear for an alpha transition makes more sense than ease-out, but I'm not sure why I did that for move also.
Also, I'm not sure why we didn't implement this with the 3D transforms, but it's likely because of lack of IE9 support.

**Comment by sebworks** [comment]
@anselmbradford, what's the deal on this?

**Comment by anselmbradford** [comment]
@sebworks Looks like the PR was abandoned. I rebased my original `3dtransform` branch to remove the `linear` transition change that @Scotchester suggested was reverted. However, I kept it for the alpha transition. This would be correct, right? A fade from transparent to opaque shouldn't change rate over time?

Anyway, if the lack of IE9 support isn't an issue happy to open a PR off of https://github.com/cfpb/cfgov-refresh/compare/3dtransforms

**Comment by sebworks** [comment]
yeah, @anselmbradford please open a new PR and close this when ready.

**Comment by anselmbradford** [comment]
Replaced with https://github.com/cfpb/cfgov-refresh/pull/2900
